common/source/docs/common-brushless-escs.rst

```diff
@@ -1,121 +1,94 @@
+- Forward motors will use normal PWM unless OneShot (see below) or :ref:`DShot <common-dshot-escs>` has been enabled
+- VTOl motors will use normal PWM when :ref:`Q_M_PWM_TYPE <Q_M_PWM_TYPE> is 0/Normal (the default)
 
-- all values sent to the ESC are checksum-protected
-- clock differences between the ESC and autopilot don't affect flight performance
-- no need to do any ESC throttle range calibration
-- very high protocol frame rates are supported
-
-While DShot is available on many ESCs, those with BLHeli32 and BLHeli-S setup capability, offer the most performance and often, additional features such as ESC telemetry. In addition, ArduPilot provides special programming capability for BLHeli32 and BLHeli-S ESCs via its passthrough feature (See :ref:`common-blheli32-passthru` section.)
-
-.. note::
-   Recently there is a growing number of proprietary and non-proprietary 16 / 32 bit ESCs with firmware that support DShot and other digital ESC protocols, but not BLHeli_32-specific features like passthrough and telemetry. See your ESC's manual for further detail on supported features.
-
-For larger aircraft with longer cable runs using DShot ESC protocol, we recommend using the lowest baud rate, DShot150, as it is the most reliable protocol (lower baudrates are less susceptible to noise on cables). Higher rates can be more susceptible to noise but also tie up the allocated DMA channel for a shorter period so can be beneficial on flight controllers with a lot of DMA sharing.
-
-For smaller craft, DShot600 is by far the most widely used and can therefore be a more suitable choice simply because of the amount of testing that it has had, rather than the newer DShot1200 protocol.
-
-Bi-directional DShot involves a longer pulse width since it has to wait for a response from the ESC before it can send another pulse and thus DShot300 and DShot600 are to be preferred. Bi-directional DShot does not share DMA channels and so there is no impact on other peripherals.
-
-.. note:: When an output is configured for DShot, the ``SERVOx_MIN/MAX/TRIM`` parameters for that output will always be ignored since DShot does not use these parameters. The trim  value used will be  1500 if it's a reversible output, or 1000 if normal output setup in DShot, and the output range always be 1000-2000. No ESC calibration step is required.
+.. note:: The autopilot should be re-booted after changing the protocol type.
+[/site]
 
-Configuration
-=============
+The frame rate of these pulses is usually between 50Hz to 490Hz. The faster frame rates allow quicker control reactions to be sent to the motor, if the ESC has capability for those frame rates.
 
-Protocol Selection
-------------------
-The ESC control protocol is selected by the :ref:`MOT_PWM_TYPE <MOT_PWM_TYPE>` parameter on Copter and Rover, or :ref:`Q_M_PWM_TYPE <Q_M_PWM_TYPE>` on QuadPlanes for its copter function motors.
+[site wiki="copter"]
+- Set :ref:`RC_SPEED <RC_SPEED>` to change the frame rate (default is 490hz)
```

Comment:
Hz

Reply:
ok, changed hz to Hz in various places. txs